### PR TITLE
chore: change missing renames of KubePrism

### PR DESCRIPTION
**Pull request by DmitriyMV**
For #7432

### CODE REVIEW

**Review by smira — COMMENTED** [review]
we had the flag in the e2e-qemu.sh, it needs to be updated

**Comment by DmitriyMV** [comment]
@smira Its [already](https://github.com/siderolabs/talos/blob/main/hack/test/e2e-qemu.sh#L135) updated.

**Comment by smira** [comment]
> @smira Its [already](https://github.com/siderolabs/talos/blob/main/hack/test/e2e-qemu.sh#L135) updated.

:shrug: how did it pass the tests then?

**Comment by DmitriyMV** [comment]
> how did it pass the tests then?

Because the flag name was updated, but flag variable name wasn't.

**Comment by smira** [comment]
> > how did it pass the tests then?
> 
> Because the flag name was updated, but the actual variable name wasn't.

got it, thanks

**Review by smira — APPROVED** [review]

File: pkg/machinery/config/generate/options.go [hunk 1]
```diff
@@ -34,10 +34,10 @@ func WithLocalAPIServerPort(port int) Option {
 	}
 }
 
-// WithAPIServerBalancerPort specifies the local API server load balancer port. If 0, lod balancer is disabled.
-func WithAPIServerBalancerPort(port int) Option {
+// WithKubePrismPort specifies the KubePrism port. If 0, lod balancer is disabled.
```
Comment (frezbo) [inline pkg/machinery/config/generate/options.go:37]:
```suggestion
// WithKubePrismPort specifies the KubePrism port. If 0, load balancer is disabled.
```

**Review by frezbo — APPROVED** [review]

File: pkg/machinery/config/generate/options.go (hunk 1 above)
Comment (DmitriyMV) [reply pkg/machinery/config/generate/options.go:37]: Lol, missed like by 0.1 second :D

Comment (DmitriyMV) [reply pkg/machinery/config/generate/options.go:37]: I'll update it in the next PR.

**Pull request merged**